src/draw/__tests__/drawRectangle.test.ts

```diff
@@ -0,0 +1,70 @@
+import { IJS } from '../../IJS';
+
+describe('we check drawRectangle', () => {
+  it('draw rectangle in an image', () => {
+    const image = testUtils.createRgbImage([
+      [100, 150, 200, 100, 150, 0],
+      [100, 200, 5, 3, 200, 0],
+      [150, 200, 255, 6, 150, 0],
+    ]);
+
+    const start = { row: 0, column: 0 };
+    const width = 2;
+    const height = 2;
+    const expected = image.drawRectangle(start, width, height, {
+      color: [255, 0, 0],
+    });
+
+    expect(expected).toMatchImageData([
+      [255, 0, 0, 255, 0, 0],
+      [255, 0, 0, 255, 0, 0],
+      [150, 200, 255, 6, 150, 0],
+    ]);
+    expect(expected).not.toBe(image);
+  });
+  it('draw rectangle with out parameter set to self', () => {
+    const image = testUtils.createRgbImage([
+      [100, 100, 200, 100, 100, 100, 150, 200, 255],
+      [100, 100, 5, 3, 100, 100, 150, 200, 255],
+      [100, 100, 255, 6, 150, 5, 5, 3, 200],
+    ]);
+
+    const start = { row: 0, column: 0 };
+    const width = 3;
+    const height = 3;
+    const expected = image.drawRectangle(start, width, height, {
+      color: [255, 0, 0],
+      fill: [0, 5, 2],
+      out: image,
+    });
+    expect(expected).toMatchImageData([
+      [255, 0, 0, 255, 0, 0, 255, 0, 0],
+      [255, 0, 0, 0, 5, 2, 255, 0, 0],
+      [255, 0, 0, 255, 0, 0, 255, 0, 0],
+    ]);
+    expect(expected).toBe(image);
+  });
+  it('draw rectangle with out parameter', () => {
+    const out = new IJS(2, 3);
+    const image = testUtils.createRgbImage([
+      [100, 150, 200, 100, 150, 0],
+      [100, 200, 5, 3, 200, 0],
+      [150, 200, 255, 6, 150, 0],
+    ]);
+    const start = { row: 1, column: 0 };
+    const width = 3;
+    const height = 3;
+    const expected = image.drawRectangle(start, width, height, {
```

Comment:
Please use only one object parameter that contains the following properties:
- row: default 0
- column: default 0
- width: default image.width
- height: default image.height
- color: default [0,0,0] (depends of the kind of image as you did)
- fill: default empty

Reply:
If default color is black how could we create rectangle with fill and no color 
(i suggest adding color "none")

Reply:
`color: 'none'` SGTM

Reply:
done